src/Microsoft.AspNetCore.Rewrite/Internal/RegexTimeoutSwitchUtility.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Rewrite.Internal
+{
+    public class RegexTimeoutSwitchUtility
+    {
+        internal static readonly string UseLowerRegexTimeoutsSwitch = "Switch.Microsoft.AspNetCore.Rewrite.UseLowerRegexTimeouts";
+        // For testing
+        public static bool UseLowerRegexTimeouts;
```

Comment:
internal. We only used instance fields for testing.

Reply:
Yours may be trickier to test as you immediately calculate a secondary value from it. Mine was testing method behavior changes so I could change an instance flag before calling the method.

Reply:
Yeah the only was I see testing the switch is through an internal bool. However, by having the timeout evaluated in the constructor, there bool wouldn't do anything. Don't see a great way to test, will remove the switch tests for now.